apps/explorer/priv/repo/migrations/20230928192434_add_index_fee_payments_to_address_hash.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Explorer.Repo.Migrations.AddIndexFeePaymentsToAddressHash do
+  use Ecto.Migration
+  @disable_ddl_transaction true
+  @disable_migration_lock true
+
+  def change do
+     create_if_not_exists(
+      index(
+        :fee_payments,
+        [:to_address_hash, "block_number DESC NULLS FIRST", "index DESC NULLS FIRST"],
```

Comment:
Why is it set to order NULL first?
Is there a scenario where block_number or index are null?

Reply:
I should get rid of that. it applies for transactions which can have null for index but it doesnt apply here.